### ISSUE TITLE
A "discard changes" button would be helpful for the "approve scenario" panel

### ISSUE THREAD

**Issue by diego2013** [diego2013/CSR_Meteor#145]
![image](https://cloud.githubusercontent.com/assets/4389491/12850592/7dd900ca-cbf3-11e5-9807-8195ef184cf9.png)

Seeing the code it looks like there was actually a button, but is commented in the code, and he functionality was never implemented

**Comment by diego2013** [comment]
Implemented this, but Router.go() is not working well from `"click #discardChanges" : function(event, template)`. If it did it would be ideal to re-render the page (force a reload to clean the form).

I had to "manually" re-render the template fields, which is not ideal.

**Comment by diego2013** [comment]
This functionality has been implemented (thus, I'm closing this issue).

The only thing left to discuss is the behavior (I'm willing to reopen this issue on user's request). Since we have a "save" button there, the "discard changes" button works as a "revert to the last saved state" and not "revert to the state before I came to this view and started modifying the scenario".

User validation would be good to know what is the most useful behavior for this feature.
